src/System.Data.Common/src/System.Data.Common.csproj

```diff
@@ -7,11 +7,10 @@
     <AssemblyVersion>4.0.1.0</AssemblyVersion>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <PlatformTarget>AnyCPU</PlatformTarget>
+    <!-- We will continue to target 1.0 so that this will work on WP8 -->
     <NuGetTargetMoniker Condition="'$(TargetGroup)' == ''">.NETStandard,Version=v1.0</NuGetTargetMoniker>
     <IsPartialFacadeAssembly Condition="'$(TargetGroup)' == 'net451'">true</IsPartialFacadeAssembly>
-    <PackageTargetFramework Condition="'$(TargetGroup)' == ''">netstandard1.2</PackageTargetFramework>
-    <PackageTargetFramework Condition="'$(TargetGroup)' == 'net451'">net451</PackageTargetFramework>
-    <NuGetTargetMoniker Condition="'$(TargetGroup)' == ''">.NETStandard,Version=v1.2</NuGetTargetMoniker>
+    <PackageTargetFramework Condition="'$(TargetGroup)' == ''">netstandard1.0</PackageTargetFramework>
```

Comment:
I'm talking about PackageTargetFramework, not NuGetTargetMoniker.  These two properties control different things.
NuGetTargetMoniker = moniker to use when resolving *this project's* nuget dependencies from project.json
PackageTargetFramework = moniker to use when packaging this project into a nuget package.